requirements-dev.txt

```diff
@@ -35,6 +35,7 @@ typed-ast==1.4.3
 typing-extensions==3.10.0.0
 types-setuptools>=57.4.17
 urllib3==1.26.9
-VapourSynth>=57
+VapourSynth>=58
 vsutil>=0.6.0
```

Comment:
Stuff you've already got in requirements.txt has no reason to be in -dev requirements

Reply:
mypy/flake8 only install this requirements.txt, not the other one.